WordPress/gutenberg
# fix: Avoid empty Gallery block error

## What?

Fixes wordpress-mobile/gutenberg-mobile#5495.

Avoid an error that occurs when the `innerBlocks` attribute is `undefined`
within a Gallery block. The error was surfaced in Sentry error reports.
https://github.com/wordpress-mobile/gutenberg-mobile/issues/5495

## Why?

The error results in the mobile editor crashing.

## How?

Setting a default function parameter avoids an error when `innerBlockImages` is
`undefined`.

This default function parameter may be unnecessary and masking a underlying
issue. It is worth investigating this circumstance further in the future.

## Testing Instructions




Verify the test instructions found in https://github.com/WordPress/gutenberg/pull/42178 succeed.

### Testing Instructions for Keyboard

n/a

## Screenshots or screencast 
n/a
